<sdf version="1.6">
   <model name="sphere_obstacle1">  
    <!--  <pose>-0.2 -0.1915 1.75 0 0 0</pose>  -->   <!-- Posição da esfera 1 (x, y, z) e rotação (roll, pitch, yaw) -->
      <static>true</static> <!-- Indica que é um objeto estático -->
      <link name="link">
         <visual name="visual">
            <geometry>
               <sphere>
                  <radius>0.05</radius>  <!-- Raio da esfera -->
               </sphere>
            </geometry>
            <material>
               <ambient>0.2 0.8 0.1 1</ambient>
               <diffuse>0.2 0.8 0.1 1</diffuse>
            </material>
         </visual>
         <collision name="collision">
             <geometry>
                <sphere>
                    <radius>0.05</radius>
               </sphere>
             </geometry>
         </collision>
      </link>
   </model>
</sdf>


